Task/Taskfile.yml

```diff
@@ -3,8 +3,127 @@
 
 version: "3"
 
+set:
+  - nounset
+  - errexit
+  - pipefail
+
 tasks:
+  lint:
+    platforms: [linux, darwin]
+    # Adjustments are passed as vars, not envs, to avoid local workarounds
+    vars:
+      INPUT_AUTO_FIX: '{{.INPUT_AUTO_FIX | default ""}}'
+      INPUT_DISABLE_MYPY: '{{.INPUT_DISABLE_MYPY | default ""}}'
+      INPUT_EXCLUDE: '{{.INPUT_EXCLUDE | default ""}}'
+      INPUT_LOG_LEVEL: '{{.INPUT_LOG_LEVEL | default ""}}'
+      PLATFORM: '{{.PLATFORM | default ""}}'
+    preconditions:
+      # Temporary workaround until we have an arm64 image on docker hub
+      - sh: |
+          if [[ "$(uname -m)" -ne "arm64" ]]; then
```

Comment:
Once we build **and push** arm64 images, then we can remove this.

Reply:
I self-assigned a task for next sprint - https://www.wrike.com/open.htm?id=1167647127